tests/test_initial_conditions.py

```diff
@@ -312,7 +312,7 @@ def processing_model(n_to_be_processed_tests, states, params, seed):
     assert df["ever_infected"].sum() == 70_000
     assert np.allclose(
         df["cd_immune_false"].value_counts(normalize=True),
-        np.array([8, 4, 2, 1]) / 15,
+        np.array([8, 7]) / 15,
         atol=1e-3,
     )
```

Comment:
Questions: Is this an implication of the new code or a sign of some bug?

Reply:
@tobiasraabe

Reply:
Just drop this test once we have removed cd_immune_false

Reply:
Do not drop it. It tests that during the burn-in phase the number of infections double each period. Replace it with another counter for the occurrence of the last infection.